projects/dev-schematics/tsconfig.json

```diff
@@ -0,0 +1,26 @@
+{
+  "extends": "../../tslint.json",
```

Comment:
Is it intenational that we extend in tsconfig tslint file?

Reply:
Oh, small omission, will be fixed with the next commit.